problems/3.conditionals.js

```diff
@@ -2,25 +2,39 @@
 * @param {number} temp - The current temperature
 * @returns {string} - A message about the temperature
 */
-function temperatureMessage() {
-
+function temperatureMessage(temp) {
+  let currentTemp = 32
+  if(temp < currentTemp) {
+    return "It's freezing!"
+  } else {
+    return "It's alright"
+  }
 }
 
 /** Returns "It's freezing!" if the temperature is below 32, and "It's alright" otherwise. The implementation must use a ternary.
 * @param {number} temp - The current temperatur
 * @returns {string} - A message about the temperature
 */
-function temperatureMessageUsingTernary() {
-
+function temperatureMessageUsingTernary(temp) {
+  return (temp ? 32 : !32);
+  return "It's freezing!"
```

Comment:
Only one return will run. This currently says if temp is truthy return 32 else return Not 32 which is false. 
We want to heck temp < 32 ? "freezing" : "warm"